src/components/common/Dialogs/NetworkContractUpgradeDialog/ContractVersionSection/ContractVersionSection.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import styles from './ContractVersionSection.css';
+
+const displayName =
+  'common.NetworkContractUpgradeDialog.NetworkContractUpgradeDialogForm.LegacyPermissionWarning';
+
+const MSG = defineMessages({
+  currentVersion: {
+    id: `${displayName}.currentVersion`,
+    defaultMessage: 'Current version',
+  },
+  newVersion: {
+    id: `${displayName}.newVersion`,
+    defaultMessage: 'New version',
+  },
+});
+
+interface Props {
+  currentVersion: number;
+}
+
+const LegacyPermissionWarning = ({ currentVersion }: Props) => {
+  const nextVersion = currentVersion + 1;
+  const networkVersion = parseInt('1', 10); // parseInt(newVersion || '1', 10);
```

Comment:
Is the comment still relevant?

Reply:
Only the `newVersion`, the rest can go :+1: